src/config.rs

```diff
@@ -11,6 +11,7 @@ use reqwest::blocking::{Client, RequestBuilder, Response};
 #[cfg(feature = "async")]
 use reqwest::{Client, RequestBuilder, Response};
 
+const DEFAULT_USER_AGENT: &str = concat!(env!("CARGO_PKG_NAME"), "/", env!("CARGO_PKG_VERSION"));
```

Comment:
Do you want this to be `pub`? Then the resulting string would also be visible in the documentation.

Reply:
But wouldn't it change when using it in an another crate? Doesn't seems good to show something that changes